srcpkgs/kicad/template

```diff
@@ -1,26 +1,26 @@
 # Template file for 'kicad'
 pkgname=kicad
-version=5.1.11
-revision=4
+version=6.0.1
+revision=1
 build_style=cmake
 build_helper=cmake-wxWidgets-gtk3
-configure_args="-DKICAD_BUILD_VERSION=${version} -DKICAD_SCRIPTING=ON
- -DKICAD_SCRIPTING_MODULES=ON -DKICAD_SCRIPTING_WXPYTHON=ON
- -DKICAD_SCRIPTING_ACTION_MENU=ON  -DBUILD_GITHUB_PLUGIN=ON -DKICAD_USE_OCE=OFF
+configure_args="-DKICAD_SCRIPTING=ON -DKICAD_SCRIPTING_MODULES=ON
+ -DKICAD_SCRIPTING_WXPYTHON=ON -DKICAD_SCRIPTING_ACTION_MENU=ON
  -DKICAD_SCRIPTING_PYTHON3=ON -DKICAD_SCRIPTING_WXPYTHON_PHOENIX=ON
  -DwxWidgets_CONFIG_EXECUTABLE=$WX_CONFIG
  -DKICAD_USE_OCC=$(vopt_if occt ON OFF) -DKICAD_SPICE=$(vopt_if spice ON OFF)"
-hostmakedepends="pkg-config swig wxWidgets-gtk3-devel python3"
+hostmakedepends="pkg-config swig wxWidgets-gtk3-devel python3 tar"
 makedepends="wxWidgets-gtk3-devel wxPython4 python3-devel glew-devel cairo-devel
- openssl-devel boost-devel libcurl-devel glm libgomp-devel
+ openssl-devel boost-devel libcurl-devel glm libgomp-devel libfreeglut-devel
```

Comment:
I think we managed to remove the dependency on OpenSSL from kicad 6, can you try removing it from here to see if it builds?

You have `libcurl-devel` repeated here.